commit dd0fc59db59d277c18117680d2592093046daff8
Author: Kirill A. Korinsky <kirill@korins.ky>
Date:   Tue Sep 21 13:15:58 2021 +0200

    Allow to configure docs path
    
    This is trivial changes allows to control docs path.

diff --git a/Docs/CMakeLists.txt b/Docs/CMakeLists.txt
index 4e9b7a608..07127893f 100644
--- a/Docs/CMakeLists.txt
+++ b/Docs/CMakeLists.txt
@@ -13,10 +13,12 @@
 ###############################################################
 
 # Set docs target directory
-if (WIN32 OR APPLE)
-  set(OGRE_DOCS_PATH "Docs")
-elseif (UNIX)
-  set(OGRE_DOCS_PATH "share/doc/OGRE")
+if (NOT OGRE_DOCS_PATH)
+    if (WIN32 OR APPLE)
+        set(OGRE_DOCS_PATH "Docs")
+    elseif (UNIX)
+        set(OGRE_DOCS_PATH "share/doc/OGRE")
+    endif ()
 endif ()
 
 # Build and install API documentation if doxygen is available
